### PR TITLE
Moved LinkedInput from addons to packages.

### PR DESCRIPTION
**Pull request by jimfb**
Moved LinkedInput from addons to packages.

### CODE REVIEW

**Comment by jimfb** [comment]
@zpao

File: packages/react-linked-input/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "react-linked-input",
+  "version": "0.0.201602251656",
```
Comment (zpao) [inline packages/react-linked-input/package.json]: Let's just make this 1.0.0

**Comment by zpao** [comment]
And github doesn't let me comment outside the context so …

Let's remove the providesModule header too

File: packages/react-linked-input/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+This component supports the ReactLink API (valueLink, etc) for input components.  Support for ReactLink on DOM elements will be removed from React.  This component may be used as a migration plan (so your code doesn't break in the next version of React) or may be used if you just like the ReactLink data binding semantics.  However, this component is not maintained, so use at your own risk.
+
+
+```
+var React = require('react');
+var ReactDOM = require('react-dom');
+var LinkedInput = require('react-linked-input');
+
+React.render(<LinkedInput valueLink={{value: 'boo', requestChange: function() {}}} />, container);
```
Comment (zpao) [inline packages/react-linked-input/README.md]: Might want to wrap this code so it looks good on npm.

**Pull request merged**